recipes/wavelet_buffer/all/conanfile.py

```diff
@@ -0,0 +1,125 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+
+import os
+
+required_conan_version = ">=1.50"
+
+
+class WaveletBufferConan(ConanFile):
+    name = "wavelet_buffer"
+    license = "MPL-2.0"
+    description = "An universal C++ compression library based on wavelet transformation"
+    topics = ("compression", "signal-processing", "wavelet")
+    homepage = "https://github.com/panda-official/WaveletBuffer"
+    url = "https://github.com/conan-io/conan-center-index/recipes/wavelet_buffer"
+    requires = "openblas/0.3.20", "blaze/3.8", "libjpeg-turbo/2.1.2", "cimg/3.0.2"
+    default_options = {
+        "cimg:enable_fftw": False,
+        "cimg:enable_jpeg": False,
+        "cimg:enable_openexr": False,
+        "cimg:enable_png": False,
+        "cimg:enable_tiff": False,
+        "cimg:enable_ffmpeg": False,
+        "cimg:enable_opencv": False,
+        "shared": False,
+        "fPIC": True,
+    }
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+
+    generators = "CMakeDeps"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+        self.settings.build_type = "Release"
```

Comment:
```suggestion
```
You can't a configuration directly, it will mask the package ID asked when running conan install. In case some configuration is not supported by the upstream, is should be excluded at `validate()`:


https://docs.conan.io/en/latest/reference/conanfile/methods.html#validate
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#the-validate-method